docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+taks a look at the subtree of the class hierarchy rooted at a
+particular superclass:
+
+.. code-block:: python
+
+  import networkx as nx
+  from networkx.drawing.nx_agraph import graphviz_layout, to_agraph
+  import pygraphviz as pgv
+  from IPython.display import Image
+
+  subtree = nx.ego_graph(hier.graph, superclass, radius=10)
+  mapping = {n: hier.HIER_NODE_NAME[n] for n in subtree.nodes()}
+  subtree = to_agraph(nx.relabel_nodes(subtree, mapping))
+  subtree.delete_edge(subtree.edges()[0])
+  subtree.layout('dot')
+  subtree.node_attr['color']='blue'
+  subtree.draw('graph.png', format='png')
+  Image('graph.png')
+  
+For instance, visualizing tree rooted at the ``fungus`` superclass yields:
+
+.. image:: ../figures/graph.png
+  :width: 600
+  :alt: Visulization of subtree rooted at a specific superclass.
+
+Part 2: Creating BREEDS Datasets
+'''''''''''''''''''''''''''''''''
+
+To create a dataset composed of superclasses, we use the 
+:class:`~robustness.tools.breeds_helpers.BreedsDatasetGenerator`.
+Internally, this class instantiates an object of 
+:class:`~robustness.tools.breeds_helpers.ClassHierarchy` and uses it
+to define the superclasses.
+
+.. code-block:: python
+
+  from robustness.tools.breeds_helpers import BreedsDatasetGenerator
+  DG = BreedsDatasetGenerator(info_dir)
+
+Specifically, we will use  
+py:meth:`~robustness.tools.breeds_helpers.BreedsDatasetGenerator.get_superclasses`.
+This function takes in the following arguments (see :meth:`this docstring
+<robustness.tools.breeds_helpers.BreedsDatasetGenerator.get_superclasses>` for more details):
+
+- :samp:`level`: Level in the hierarchy (in terms of distance from the
+  root node) at which to define superclasses.
+- :samp:`Nsubclasses`: Controls the minimum number of subclasses/superclass
+  in the dataset. If None, it is automatically set to be the size (in terms
+  of subclasses) of the smallest superclass. 
+- :samp:`split`: If ``None``, subclasses of a superclass are returned 
+  as is, without partitioning them into the source and target domains. 
+  Else, can be ``rand/good/bad`` depending on whether the subclass split should be
+  random or less/more adversarially chosen [STM20]_.
+- :samp:`ancestor`: If a node ID is specified, superclasses are chosen from 
+  subtree of class hierarchy rooted at this node. Else, if None, :samp:`ancestor`
+  is set to be the root node.
+- :samp:`balanced`: If True, subclasses/superclass is fixed over superclasses.
+
+For instance, we could create a balanced dataset, with the subclass partition 
+being less adversarial as follows:
+
+.. code-block:: python
+
+   ret = DG.get_superclasses(level=2, 
+                          Nsubclasses=None, 
+                          split="rand", 
+                          ancestor=None, 
+                          balanced=True)
+  subclass_ranges, label_map, subclass_tuple, superclasses, _ = ret                                    
+
+This method returns:
```

Comment:
- List the return values below in the same order they are returned
- What is the `_` in the return value? 
- What is the point of `subclass_ranges`, isn't it just a straightforward transformation of `subclass_tuple`
- Rename `subclass_tuple` to like `subclass_splits` or something more descriptive IMO

Reply:
Fixed and reduced returns.